App/Application/AppNavigation.swift

```diff
@@ -250,10 +249,16 @@ extension TabbarVC: RoutableWithConfiguration {
 
 // MARK: SensitiveDataCover
 
-extension SensitiveDataCoverVC: Routable {
+extension SensitiveDataCoverVC: RoutableWithConfiguration {
   var routeIdentifier: RouteElementIdentifier {
     return Screen.sensitiveDataCover.rawValue
   }
+
+  var navigationConfiguration: [NavigationRequest: NavigationInstruction] {
+    return [
+      .hide(Screen.sensitiveDataCover): .dismissModally(behaviour: .soft),
```

Comment:
<!--
  0 failure: 
  1 warning:  swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: App/Application/AppNavigation.swift;
  Line: 259;
-->

- :warning: swiftlint (0.39.2): Collection literals should not have trailing commas. (trailing_comma)